src/main/java/com/google/sps/servlets/ShowInterviewersServlet.java

```diff
@@ -0,0 +1,162 @@
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.common.base.Preconditions;
+import com.google.gson.Gson;
+import com.google.sps.data.Availability;
+import com.google.sps.data.AvailabilityDao;
+import com.google.sps.data.DatastoreAvailabilityDao;
+import com.google.sps.data.DatastorePersonDao;
+import com.google.sps.data.DatastoreScheduledInterviewDao;
+import com.google.sps.data.Person;
+import com.google.sps.data.PersonDao;
+import com.google.sps.data.PossibleInterviewer;
+import com.google.sps.data.ScheduledInterview;
+import com.google.sps.data.ScheduledInterviewDao;
+import com.google.sps.data.TimeRange;
+import java.io.IOException;
+import java.io.BufferedReader;
+import java.lang.Integer;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.TextStyle;
+import java.util.Locale;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.HashSet;
+import java.util.Set;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+import javax.servlet.RequestDispatcher;
+import java.util.Optional;
+
+@WebServlet("/show-interviewers")
+public class ShowInterviewersServlet extends HttpServlet {
+
+  private AvailabilityDao availabilityDao;
+  private PersonDao personDao;
+
+  @Override
+  public void init() {
+    init(new DatastoreAvailabilityDao(), new DatastorePersonDao());
+  }
+
+  public void init(AvailabilityDao availabilityDao, PersonDao personDao) {
+    this.availabilityDao = availabilityDao;
+    this.personDao = personDao;
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String utc = request.getParameter("utc");
+    Instant startOfRange = Instant.parse(utc);
+    Instant endOfRange = Instant.parse(utc).plus(1, ChronoUnit.HOURS);
+    List<Availability> availabilitiesInRange =
+        availabilityDao.getInRangeForAll(startOfRange, endOfRange);
+    List<Person> possiblePeople =
+        getPossiblePeople(availabilitiesInRange, startOfRange, endOfRange);
+    Set<PossibleInterviewer> possibleInterviewers = getPossibleInterviewers(possiblePeople);
```

Comment:
This is confusing bc it seems like `possiblePeople` and `possibleInterviewers` are different, but `possibleInterviewers` is just the Set version of `possiblePeople`. `getPossiblePeople()` can just return a Set instead of a List and then converting it back into the Set it basically was originally.

Reply:
They are different, notice the classes / class names. Person is different from PossibleInterviewer. I agree the naming of those variable is kind of unfortunate though. Open to suggestions.

Reply:
Changed getPossibleInterviewers to peopleToPossibleInterviewers as discussed.